### ISSUE TITLE
DMARC policy mail.ru

### ISSUE THREAD

**Issue by SerhiyP** [agragregra/uniMail#6]
This  message was not accepted due to domain owner DMARC policy

**Comment by jutout** [comment]
Здравствуйте, есть маленькая проблемка. 
С Вашим маил скриптом 
http://webdesign-master.ru/blog/tools/578.html

Изначально ваш скрипт работал все прекрасно. 
Но потребовалось добавить Аякс и Автозаполнение для поиска. 
И тут происходит конфликт. 
По раздельности они работают а вместе нет. кнопка Сабмит на поиске не срабатывает. Я так думаю что это из за того что в JS используется два раза запрос АЯКС. 
Так как я в JS не шарю , то прошу помощи. 
Вот этот поиск http://www.smarttutorials.net/jquery-autocomplete-sea.. 

Буду благодарен за подсказку. 

<pre> 
// Поиск AJAX autocomplete 
$('#medic_name').autocomplete({ 
source: function( request, response ) { 
$.ajax({ 
url : 'search/ajax.php', 
dataType: "json", 
data: { 
name_startsWith: request.term, 
type: 'country' 
}, 
success: function( data ) { 
response( $.map( data, function( item ) { 
return { 
label: item, 
value: item 
} 
})); 
} 
}); 
}, 
autoFocus: true, 
minLength: 1 
}); 
</pre> 

<pre> 
$(document).ready(function() { 

//E-mail Ajax Send 
$("form").submit(function() { //Change 
var th = $(this); 
$.ajax({ 
type: "POST", 
url: "../blocks/mail_uni.php", //Change 
data: th.serialize() 
}).done(function() { 
alert("Спасибо Ваше сообщение отправленно!"); 
setTimeout(function() { 
// Done Functions 
th.trigger("reset"); 
}, 1000); 
}); 
return false; 
}); 

}); 
</pre>

**Comment by AndryWJ** [comment]
Хотел бы добавить что если форма в модальном окне, то есть появляется после загрузки страницы, то работать будет только если вешать так :
	//E-mail Ajax Send
	$(document).on('submit','#form', function(){
		var th = $(this);
		$.ajax({
			type: "POST",
			url: "mail.php", //Change
			data: th.serialize()
		}).done(function() {
                        alert("Send done");
			setTimeout(function() {
				// Done Functions
				th.trigger("reset");
			}, 1000);
		});
		return false;
	});
